pkgs/applications/virtualization/xen/4.15.nix

```diff
@@ -125,7 +125,7 @@ callPackage (import ./generic.nix (rec {
     ++ optional (withSeabios) "--with-system-seabios=${seabios}/share/seabios"
     ++ optional (!withInternalSeabios && !withSeabios) "--disable-seabios"
 
-    ++ optional (withOVMF) "--with-system-ovmf=${OVMF.fd}/FV/OVMF.fd"
+    ++ optional (withOVMF) "--with-system-ovmf=${OVMF.firmware}"
```

Comment:
This is a different file.
```
nix-repl> OVMF.firmware
"/nix/store/jhnng8aqgw824a08ly9h878rzzgm5m92-OVMF-202402-fd/FV/OVMF_CODE.fd"
```

The other day I tried to boot a USB stick in `qemu`, but it didn't work with this attribute; I had to use `OVMF.fd`, not `OVMF_CODE.fd`.

Maybe this use case is different; I'm certainly no expert on this, but it looks wrong.

Reply:
You were probably using `-bios OVMF.fd`, which is not entirely correct, you actually need to use the `OVMF_CODE.fd` and `OVMF_VARS.fd` files separately. AFAICT Xen should do the right thing here, though I haven't tested it directly.